### PR TITLE
Add home page doc support for readme and getting started

### PR DESCRIPTION
**Pull request by nwmac**
The home page has a couple of small docs that need to be included - this PR adds support for those using nuxt content.

The PR ensures that i18n is supported for these docs, with a fallback to en-us.

### CODE REVIEW

**Review by richard-cox — CHANGES_REQUESTED** [review]

File: content/docs/en-us/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: Getting Started
+---
+# Gettting Started
+
+Welcome to Rancher Dashboard.
```
[the remark this answers is not in the document]
Comment (nwmac) [reply content/docs/en-us/getting-started.md:6]: I'm going to ask someone to help us build that out

File: pages/home/index.vue [hunk 2]
```diff
@@ -187,7 +185,13 @@ export default {
     },
 
     showWhatsNew() {
-      this.$router.push({ name: 'release-notes' });
+      // Update the value, so that the message goes away
```
[the remark this answers is not in the document]
Comment (nwmac) [reply pages/home/index.vue:188]: I'll make version number a link - I am going to tweak the behaviour slightly in another PR

File: pages/home/index.vue [hunk 3]
```diff
@@ -224,14 +228,15 @@ export default {
...
+            :title="t('landing.gettingStarted.title')"
             :pref="HIDE_HOME_PAGE_CARDS"
             pref-key="migrationTip"
           >
-            {{ t('landing.migration.body') }}
+            {{ t('landing.gettingStarted.body') }}
             <br />
-            <!-- Unhide this when docs are added -->
-            <a class="hide pull-right" href="#">{{ t('landing.learnMore') }}</a>
+            <nuxt-link :to="{name: 'docs-doc', params: {doc: 'getting-started'}}" class="pull-right">
```
[the remark this answers is not in the document]
Comment (nwmac) [reply pages/home/index.vue:237]: Fixed

**Comment by nwmac** [comment]
@richard-cox Updated following your review

**Review by richard-cox — APPROVED** [review]
Given `getting-started.md` gets updated before release, LGTM

**Pull request merged**